misc/services/mobile-webui/mobile-webui-frontend/src/__tests__/utils/qrCode/common.test.js

```diff
@@ -4,42 +4,102 @@ describe('common tests', () => {
   describe('isBarcodeProductNoMatching', () => {
     it('expectedProductNo is null', () => {
       expect(
-        isBarcodeProductNoMatching({ expectedProductNo: null, barcodeProductNo: '12345', barcodeType: 'HU' })
+        isBarcodeProductNoMatching({
+          expectedProductNo: null,
+          expectedEAN13ProductCode: null,
+          barcodeProductNo: '12345',
+          barcodeType: 'HU',
+        })
       ).toBe(true);
     });
     it('barcodeProductNo is null', () => {
       expect(
-        isBarcodeProductNoMatching({ expectedProductNo: '12345', barcodeProductNo: null, barcodeType: 'HU' })
+        isBarcodeProductNoMatching({
+          expectedProductNo: '12345',
+          expectedEAN13ProductCode: null,
+          barcodeProductNo: null,
+          barcodeType: 'HU',
+        })
       ).toBe(true);
     });
     it('HU barcode - exact product', () => {
       expect(
-        isBarcodeProductNoMatching({ expectedProductNo: '12345', barcodeProductNo: '12345', barcodeType: 'HU' })
+        isBarcodeProductNoMatching({
+          expectedProductNo: '12345',
+          expectedEAN13ProductCode: null,
```

Comment:
why adding `expectedEAN13ProductCode: null` on all those tests?
IMHO it's not needed